x-pack/plugins/snapshot_restore/public/application/sections/home/snapshot_list/snapshot_list.tsx

```diff
@@ -290,41 +257,41 @@ export const SnapshotList: React.FunctionComponent<RouteComponentProps<MatchPara
       />
     );
   } else {
-    const repositoryErrorsWarning = Object.keys(errors).length ? (
-      <Fragment>
-        <EuiCallOut
-          title={
-            <FormattedMessage
-              id="xpack.snapshotRestore.repositoryWarningTitle"
-              defaultMessage="Some repositories contain errors"
-            />
-          }
-          color="warning"
-          iconType="alert"
-        >
-          <FormattedMessage
-            id="xpack.snapshotRestore.repositoryWarningDescription"
-            defaultMessage="Snapshots might load slowly. Go to {repositoryLink} to fix the errors."
-            values={{
-              repositoryLink: (
-                <EuiLink {...reactRouterNavigate(history, linkToRepositories())}>
-                  <FormattedMessage
-                    id="xpack.snapshotRestore.repositoryWarningLinkText"
-                    defaultMessage="Repositories"
-                  />
-                </EuiLink>
-              ),
-            }}
-          />
-        </EuiCallOut>
-        <EuiSpacer />
-      </Fragment>
-    ) : null;
-
     content = (
-      <Fragment>
-        {repositoryErrorsWarning}
-
+      <>
+        {snapshots.length === SNAPSHOT_LIST_MAX_SIZE && (
+          <>
+            <EuiCallOut
+              color="warning"
+              iconType="help"
+              data-test-subj="maxSnapshotsWarning"
+              title={i18n.translate(
+                'xpack.snapshotRestore.snapshotsList.maxSnapshotsDisplayedTitle',
+                { defaultMessage: 'Cannot show the full list of snapshots' }
+              )}
+            >
+              <FormattedMessage
+                id="xpack.snapshotRestore.snapshotsList.maxSnapshotsDisplayedDescription"
+                defaultMessage="You've reached the maximum number of snapshots. To view all of your snapshots, use {docLink}."
```

Comment:
@lockewritesdocs Does "You've reached the maximum number of snapshots" sound like you've hit a technical limit, and you can't have more snapshots beyond this point? I think something like "You've reached the maximum number of snapshots that can be displayed" will remove this ambiguity. WDYT?

Also, do you think it'd be helpful to mention what that limit is, somehow?

Reply:
We could say something like, "You've reached the maximum number of viewable snapshots." 

I'm debating whether the limit is useful. If it's 500 or it's 1,000, does it matter if I can't see them? I'm not entirely sure.

Reply:
I've changed this to: "You've reached the maximum number of viewable snapshots." I'm not sure if the limit is useful either. I think it's also somewhat obvious based on the pagination.

Reply:
++ @alisonelizabeth -- I think this is a great compromise.